src/annotator/delegator.js

```diff
@@ -31,13 +31,18 @@ export default class Delegator {
    * @param {Object} [config]
    */
   constructor(element, config) {
-    // Some subclasses rely on a legacy mechanism of defining options at
-    // the class level by defining an `options` property on the prototype.
-    const classOptions = this.options || {};
-    this.options = { ...classOptions, ...config };
-    this.element = $(element);
+    this.options = { ...config };
+    this.element = element;
 
-    this.on = this.subscribe;
+    const el = /** @type {any} */ (element);
```

Comment:
Given that `element` is `{HTMLElement}` and mandatory, can this really have an `{any}` type?

Reply:
The typecast is used here so we can set the `_hypothesisEventBus` custom property. You can typecast anything to "any", but you won't get help from TS with respect to correctness for anything you do with an `any` value.